src/library_webgl.js

```diff
@@ -1867,8 +1902,13 @@ var LibraryGL = {
 #if GL_ASSERTIONS
     GL.validateGLObjectID(GL.timerQueriesEXT, id, 'glGetQueryObjecti64vEXT', 'id');
 #endif
+#if USE_WEBGL2
+    var query = GL.queries[id];
+    var param = GLctx['getQueryParameter'](query, pname);
```

Comment:
For USE_WEBGL2 + webgl1, getQueryParameter wouldn't be present here.

Reply:
Done -- this is a runtime check now.